src/WarcraftLegacies.Source/Factions/Skywall.cs

```diff
@@ -1,29 +1,77 @@
-﻿using MacroTools.FactionSystem;
+﻿using MacroTools.Extensions;
+using MacroTools.FactionSystem;
+using MacroTools.PassiveAbilities;
+using MacroTools.PassiveAbilitySystem;
+using MacroTools.Powers;
+using MacroTools.ResearchSystems;
+using MacroTools.Spells;
+using MacroTools.SpellSystem;
 using WarcraftLegacies.Shared.FactionObjectLimits;
+using WarcraftLegacies.Source.Researches;
 using WarcraftLegacies.Source.Setup;
+using WCSharp.Shared.Data;
 
 namespace WarcraftLegacies.Source.Factions
 {
   public sealed class Skywall : Faction
   {
     /// <inheritdoc />
-    public Skywall() : base("Elemental Lords", PLAYER_COLOR_PEACH, "|c00540081",
+    public Skywall() : base("Elemental Lords", PLAYER_COLOR_LIGHT_GRAY, "|cffffffff",
```

Comment:
Really no point having the class be called Skywall and the actual faction named Elemental Lords. Pick one and keep it the same for:
 - PlayerNumbers.txt
 - Class names
 - The faction name
 - Editor suffixes (if in use)